### PR TITLE
LPD-22595 Add the alert to VisualizationModes and remove from Cards and List

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPD-22595

I added the Alert to `VisualizationModes.tsx` and removed it from Cards.tsx and LIst.tsx so that the Info is available to all tabs.
Please let me know if there are any questions or comments about this.

Thank you!

### CODE REVIEW

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx [hunk 1]
```diff
@@ -70,7 +71,21 @@ export default function VisualizationModes(props: IFDSViewSectionProps) {
...
+									<ClayLayout.ContentCol className="c-gap-4 cards-visualization-mode">
+										<ClayAlert
+											displayType="info"
+											title={`${Liferay.Language.get(
+												'info'
+											)}:`}
+											variant="stripe"
+										>
+											{Liferay.Language.get(
+												'this-visualization-mode-will-not-be-shown-until-you-assign-at-least-one-field-to-a-card-element'
```
Comment (juanjofgliferay) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx:83]: This is the Cards info message. It does not make sense for all different visualization modes.

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx [hunk 2]
```diff
@@ -70,7 +71,21 @@ export default function VisualizationModes(props: IFDSViewSectionProps) {
 									className="px-0"
 									key={visualizationMode.visualizationModeId}
 								>
-									<Component {...props} />
+									<ClayLayout.ContentCol className="c-gap-4 cards-visualization-mode">
```
Comment (juanjofgliferay) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx:74]: The `cards-visualization-mode` className is used specifically to check cards container in Playwright tests. We should not reuse it here.

**Comment by fortunatomaldonado** [comment]
Thank you @juanjofgliferay for pointing out the changes that need to be made.

I am reaching out to Gerardo Vázquez to get the expected Info Message for this or if it is necessary.
Thank you for the help!

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
Hello @juanjofgliferay 

I made changes to the original fix and now have a specific info message for the Table tab.
Please let me know if there should be any changes made.

Thank you!

**Comment by juanjofgliferay** [comment]
@fortunatomaldonado Final UI looks nice. However, I think that we should follow the same pattern for the alert in all cases.
We have 3 different types of visualization: Table, List and Cards and, all of them are children of the `VisualizationModes` component. With the current solution, we have `alert` for List and Cards and use a condition to display the alert in the Table but, from the parent component.

So, you have a couple of options:

## Option 1
You can move your code to the `Table` component and wrap the [`OrderableTable`](https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx#L935) with it. This way, all three components have the same structure.

## Option 2
Use the actual code in `VisualizationModes` to conditionally display the message based on the visualization mode and remove the Alert from List and Cards. You can also change the `className` of the wrapper based on the label

```
<ClayLayout.ContentCol
        className={`c-gap-4 ${visualizationMode.label.toLowerCase()}-visualization-mode`}>
	<ClayAlert
		displayType="info"
		title={`${Liferay.Language.get(
			'info'
		)}:`}
		variant="stripe"
	>
		{visualizationMode.label === 'Table' && Liferay.Language.get(
			.... table message
		)}

		{visualizationMode.label === 'Table' && Liferay.Language.get(
			.... list message
		)}

		{visualizationMode.label === 'Table' && Liferay.Language.get(
			.... cards message
		)}
	</ClayAlert>

	<Component {...props} />
</ClayLayout.ContentCol>
```

**Comment by fortunatomaldonado** [comment]
Thank you @juanjofgliferay!

I decided to go with Option 1 as I found it to be more readable than Option 2.
Please let me know if theres any more changes needed.

Thank you!

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Review by juanjofgliferay — APPROVED** [review]
LGTM :rocket:

**Comment by fortunatomaldonado** [comment]
ci:forward

**Pull request closed without merging**